### ISSUE TITLE
Cannot open .heic files on Windows 10 Pro 1703 64 bit

### ISSUE THREAD

**Issue by miaocanada** [liuziangexit/HEIF-Utility#2]
It seems like under Windows 10 1703 (creators update) 64 bit the utility program cannot open any .heic files, even if Visual C++ 2015 redistributable is installed. Suggested testing/keeping compatibility on the following Windows Platforms (see below).

标题： 无法在Windows 10 专业版 1703 64位系统打开 .heic 文件

内容： 貌似该应用程序在Windows 10 1703 （创造者更新） 64位系统下无法打开任何的 .heic 文件，即便Visual C++ 2015 redistributable 已经安装在了系统上。建议在以下的Windows平台上进行该应用程序的测试，或者保证应用程序在以下平台的兼容性（见下）。

32 and 64 bit of 
32位和64位的
Windows Vista SP2 (NT 6.0) [1]
Windows 7 SP1 (NT 6.1)
Windows 8 (NT 6.2) [1]
Windows 8.1 (NT 6.3)
Windows 10 1507 (NT 10.0.10240) [2]
Windows 10 1511 (NT 10.0.10586)
Windows 10 1607 (NT 10.0.14393)
Windows 10 1703 (NT 10.0.15063)

[1] Microsoft no longer support these systems 微软已经不再为这些系统提供支援
[2] Although publicly announced as "not supported", Microsoft is still providing updates to these systems 尽管公开声称不再支援这些系统，微软仍然为这些系统提供更新

**Comment by liuziangexit** [comment]
谢谢您的反馈，我将会在明天对此问题进行测试。
Thank you for your feedback, and I will test this question tomorrow.

**Comment by liuziangexit** [comment]
emmm...do you mean my HEIF Utility cant open heic file?or you mean Windows 10 it self cant open heic?

**Comment by miaocanada** [comment]
I mean your utility program. It just bumps me a message box saying "cannot open this file" in Chinese, when I try to open the file by either opening the drop down menu, or dragging and dropping the file onto the program window. Files that I used are from the sample folder.

Edit: I've edited the Chinese translation on the main description of this issue ticket. I'm thinking that could cause the confusion... Sorry about that.

**Comment by liuziangexit** [comment]
i dont know why cause that...i and my friend tested it in Windows 7 and 10,but it works good...maybe you should try to run it as administrator?cause program will create temp file in folder,maybe it dont has permission to write?

**Comment by miaocanada** [comment]
Alright I've discovered what the problem is: the utility program requires an **x86** version of Visual C++ 2015 redistributable, since this utility is an **x86** program. My machine has only **x64** version of that redistributable installed. It does not matter what Windows operating system you use, nor do the privileges (administrator etc.).

https://www.microsoft.com/en-ca/download/details.aspx?id=48145
Basically just download the x86 version from the above link, install it on your machine and this program works.

If applicable you can update the instruction of github or your website, and probably close this issue ticket.

Sorry for that.

**Comment by liuziangexit** [comment]
lol, thank you for your help, i will release a english version today.you can download it later :)
